packages/graphql/src/services/GraphQLContext.js

```diff
@@ -6,15 +6,20 @@ import createItinerariesLoader, {
   type ItinerariesSearchParameters,
   type Itineraries,
 } from '../dataloaders/Itinerariesloader';
+import createLocationLoader, {
+  type Locations,
+} from '../dataloaders/Locationsloader';
 
 export type GraphqlContextType = {|
   +dataLoader: {|
     +itineraries: DataLoader<ItinerariesSearchParameters, Itineraries[]>,
+    +locations: DataLoader<{ term: string }, Locations>,
```

Comment:
type TermInput = {| +term: string |}

And I am pretty sure it should return Locations[], not Locations.

Reply:
'Locations' are already typed as an array https://github.com/kiwicom/margarita/blob/f89766afccf368c949d10118e6d844395a235191/packages/graphql/src/dataloaders/Locationsloader.js#L23